src/main/java/au/com/rayh/XCodeBuilder.java

```diff
@@ -540,16 +609,16 @@ private boolean _perform(Run<?,?> build, FilePath projectRoot, Launcher launcher
         /*returnCode =*/ launcher.launch().envs(envs).cmds("/usr/bin/security", "find-identity", "-p", "codesigning", "-v").stdout(listener).pwd(projectRoot).join();
 
         Team team = getDevelopmentTeam();
-        if(team == null)
-        {
-            listener.fatalError(Messages.XCodeBuilder_teamNotConfigured());
-            return false;
-        }
-        String developmentTeamID = envs.expand(team.getTeamID());
-        if (!StringUtils.isEmpty(developmentTeamID)) {
-            listener.getLogger().println(Messages.XCodeBuilder_DebugInfoCanFindPProfile());
-            /*returnCode =*/ launcher.launch().envs(envs).cmds("/usr/bin/security", "find-certificate", "-a", "-c", developmentTeamID, "-Z", "|", "grep", "^SHA-1").stdout(listener).pwd(projectRoot).join();
-            // We could fail here, but this doesn't seem to work as it should right now (output not properly redirected. We might need a parser)
+        if (team == null) {
+            listener.getLogger().println(Messages.XCodeBuilder_teamNotConfigured());
+        } else {
+            String developmentTeamID = envs.expand(team.getTeamID());
```

Comment:
This change forces "developmentTeamID" into a reduced scope, causing it to appear as empty in the following lines, namely the lines exporting it to the ExportOptions.plist, causing the plugin to fail building...